library/formsoptions.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/*
+ * Functions to be used in the forms.
```

Comment:
I suppose so.  I was just suggestion of "stuff for encounter forms, just common functions, don't put any BS in here or I will bomb.gzip your butt."
It is fine, and I guess I am just wasting time.  This stuff is ready to be merged...Oh Third set of Eyes! 👁‍🗨 👁